DataFormats/EcalDigi/interface/EcalLiteDTUSample.h

```diff
@@ -0,0 +1,47 @@
+#ifndef DataFormats_EcalDigi_EcalLiteDTUSample_h
+#define DataFormats_EcalDigi_EcalLiteDTUSample_h
+
+#include <iosfwd>
+#include <cstdint>
+#include "CondFormats/EcalObjects/interface/EcalConstants.h"
```

Comment:
this dependency is not allowed in DataFormats packages.

Reply:
Do you mean EcalConstants? It is used to have all the parameters of ecal reco in a single place, because in the legacy there are hard coded values in many places. 
Do you have a suggestion to fix it?

Reply:
I see. EcalConstants is included in the following classes. Moving the header in the DataFormat package is ok for all? If it is i move it right now.
CalibFormats/CaloObjects/src/CaloTSamples.cc
CondFormats/EcalObjects/interface/EcalLiteDTUPedestals.h
DataFormats/EcalDigi/interface/EcalDataFrame_Ph2.h
DataFormats/EcalDigi/interface/EcalDigiCollections.h
DataFormats/EcalDigi/interface/EcalLiteDTUSample.h
SimCalorimetry/EcalSimAlgos/interface/EBHitResponse.h
SimCalorimetry/EcalSimAlgos/interface/EcalCorrelatedNoiseMatrix.h
SimCalorimetry/EcalSimAlgos/interface/EcalHitResponse.h
SimCalorimetry/EcalSimAlgos/interface/EcalLiteDTUCoder.h
SimCalorimetry/EcalSimProducers/plugins/EcalLiteDTUPedestalsESProducer.cc

Reply:
Moved to DataFormats/EcalDigi/interface/EcalConstants.h https://github.com/cms-sw/cmssw/pull/31726/files#diff-147b96bd56dfbec920a69ea6826282998fa64f5c2db1275265584fb5a7c3a9fd